plugins/azure-functions-backend/src/service/router.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { errorHandler } from '@backstage/backend-common';
+import express from 'express';
+import Router from 'express-promise-router';
+import { Logger } from 'winston';
+
+import { AzureWebManagementApi } from '../api';
+
+/** @public */
+export interface RouterOptions {
+  logger: Logger;
+  azureWebManagementApi: AzureWebManagementApi;
+}
+
+/** @public */
+export async function createRouter(
+  options: RouterOptions,
+): Promise<express.Router> {
+  const { logger, azureWebManagementApi } = options;
+
+  const router = Router();
+  router.use(express.json());
+
+  router.get('/health', (_, response) => {
+    logger.info('PONG!');
+    response.send({ status: 'ok' });
+  });
+  router.post('/list', async (request, response) => {
```

Comment:
Would it not make more sense to have this as a GET? For example: `router.get('/list/:functionName', async (request, response)`. Then you can get the function name from the `request`: `const { functionName } = request.params;`